### PR TITLE
Adding Brazilian Portuguese settings to innersourcecommons.org

### PR DESCRIPTION
**Pull request by yuhattor**
Added Brazilian Portuguese settings to innersourcecommons.org 

```Related PR```: https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/549

Hi @fer-correa and @ArmandoGuimaraes, 
Could you check if the machine translation of below files is okay or not? If not, please send PR to ```adding-pt-br-language``` branch:)

- [ ] config/_default/menus.pt-br.yaml
- [ ] i18n/pt-br.yaml

@rrrutledge,
- [ ] Since merging the LearningPath PR now won't break the innersourcecommons.org page, can you have it merged into main or master for now?

So far it is as follows. This screenshot shows a temporary environment that you can easily reproduce on GitHub Codespaces.
1. fork the ```adding-pt-br-language``` branch 
2. Open it on GitHub Codespaces
3. Run the following command in the terminal in your codespace

```bash
. /i18n.sh copy # And please type "Yes"
docker compose up -d
```

4. ```pt-br``` env should be only shown under ```/learn/learning-path``` URL after [the PR](https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/549) is merged on LearningPath repository

image

### CODE REVIEW

File: config/_default/languages.yaml [hunk 1]
```diff
@@ -29,4 +29,8 @@ ru:
 fr:
   languageName: Français
   contentDir: content/fr
+  weight: 3
+pt-br:
+  languageName: Brazilian Portuguese
```
Comment (rrrutledge) [inline config/_default/languages.yaml]: @ArmandoGuimaraes is there a localized name that can be used here?

**Review by rrrutledge — APPROVED** [review]
Thank you for this work, @yuhattor!

It would be great to create a new section at [i18n.md](https://github.com/InnerSourceCommons/innersourcecommons.org/blob/master/i18n.md) - **How to add a new language to innersourcecommons.org**.

**Comment by rrrutledge** [comment]
@yuhattor I created the [adding-pt-br-language-with-learning-path](https://github.com/InnerSourceCommons/innersourcecommons.org/tree/adding-pt-br-language-with-learning-path) branch (with codespace) that has your file and also the Brazilian Portuguese learning path files.  Everything looks right to me, but for some reason I don't see the translated files showing up on the website.  I haven't tried to debug what is happening but just wanted to have you take a look at that branch to see if you notice anything off?  Thank you!

File: config/_default/languages.yaml (hunk 1 above)
Comment (fer-correa) [reply config/_default/languages.yaml]: @rrrutledge , any of these options:

- Português Brasileiro
- Português do Brasil

Comment (rrrutledge) [inline config/_default/languages.yaml]:
```suggestion
  languageName: Português Brasileiro
```

**Comment by yuhattor** [comment]
@rrrutledge @fer-correa 
Could you review it and confirm if it's mergeable?

## How to review

1. Go to [this PR](https://github.com/InnerSourceCommons/innersourcecommons.org/pull/525) page and open the environment with GitHub Codespaces
image

2. Run the command to obtain a local copy of the necessary files
```sh
./i18n.sh copy  
```
image

3. Run hugo with the command
```sh
docker compose up -d
```
image

4. Open the page in the browser
image

5. Check that nothing changed in the top page
image

6. Check that you can only select Brazilian Portuguese under ```/learn/learning-path/```
image

7. Check that you can see the language menu with English contents (Once learning-path is published, there will be the Brazilian Portuguese contents

image

**Review by rrrutledge — APPROVED** [review]
Looks (and works) great!  Thank you, @yuhattor ❗

**Review by fer-correa — APPROVED** [review]
Thank you, @yuhattor ! It's perfect 👍

**Comment by yuhattor** [comment]
@voborgus could you approve it?

**Review by voborgus — APPROVED** [review]
awesome job 🎉

**Comment by voborgus** [comment]
The language will appear in the learning path section when the Portuguese content will be merged to the master branch.

**Comment by yuhattor** [comment]
Thanks!! @voborgus

**Pull request merged**